pipelines/release.yml

```diff
@@ -1295,6 +1296,24 @@ jobs:
       repository: bumped-repo
       merge: true
 
+- name: pin-resource-type-versions
+  serial: true
+  plan:
+  - in_parallel:
+    - get: linux-rc
+      passed: [shipit]
+      trigger: true
+    - get: ci
+  - task: extract-resource-type-versions
+    file: ci/tasks/extract-resource-type-versions.yml
+  - task: pin-resource-type-versions
+    file: ci/tasks/pin-resource-type-versions.yml
+    params:
+      RELEASE_MINOR: ((release_minor))
+    output_mapping: {ci: bumped-ci}
+  - put: ci
+    params: {repository: bumped-ci}
```

Comment:
@aoldershaw looks like the private key in vault doesn't have push permissions: https://ci.concourse-ci.org/teams/main/pipelines/release/jobs/pin-resource-type-versions/builds/3?vars.version=%227.2.x%22

Reply:
Thanks, fixed now